src/apps/main/home/index.js

```diff
@@ -1,20 +1,149 @@
 import { setState } from '/store'
+import Avatar from '@app-elements/avatar'
+import Carousel from '@app-elements/carousel'
+import Dropdown from '@app-elements/dropdown'
+import Image from '@app-elements/image'
+import LoadingIndicator from '@app-elements/loading-indicator'
+import Tooltip from '@app-elements/tooltip'
+import { showNotification } from '@app-elements/notification'
+
+import { Checkbox } from '/elements/checkbox'
+import { ElementHolder } from '/elements/element-holder'
+import { Radio } from '/elements/radio'
+import { Select } from '/elements/select'
+import { TextArea } from '/elements/textarea'
+import { TextInput } from '/elements/text-input'
+
+const MyTrigger = ({ children, ...props }) =>
+  <div {...props}>{children}</div>
 
 export default function Home () {
   const openModal = (ev) => {
     ev.preventDefault()
     setState({ modal: 'ExampleModal' })
   }
   return (
-    <div>
-      <h1>PWA Starter</h1>
-      <p>
-        <button onClick={openModal}>Open Modal</button>
-      </p>
-      <img
-        src='https://camo.githubusercontent.com/974ab5e63b58eb0bb51ca21eb1b047e64ee0ea6d/68747470733a2f2f692e696d6775722e636f6d2f6b4a37673457472e6a7067'
-        alt='PWA Starter'
-      />
+    <div className='container'>
+
+      <div className='elements-wrapper'>
+
+        <div className='legend'>
+          <ul>
+            <li><a data-external-link href='#Avatar'>Avatar</a></li>
+            <li><a data-external-link href='#Button'>Button</a></li>
+            <li><a data-external-link href='#Carousel'>Carousel</a></li>
+            <li><a data-external-link href='#Dropdown'>Dropdown</a></li>
+            <li><a data-external-link href='#Form'>Form</a></li>
+            <li><a data-external-link href='#Loading'>Loading</a></li>
+            <li><a data-external-link href='#Modal'>Modal</a></li>
+            <li><a data-external-link href='#Notification'>Notification</a></li>
+            <li><a data-external-link href='#Tooltip'>Tooltip</a></li>
```

Comment:
@staydecent All the changes you suggested were np.

The active state on anchors is the only one that feels odd. I added it with hooks.

It could be done better I bet, where it actually matches the url, but I digress.

Not sure it is worth re-doing, I could just add these as routes and use the Link element.